src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -512,7 +515,22 @@ private static string GenerateUpgradeReport(NuGetProject nuGetProject, string ba
                         packageCount,
                         duration.TotalSeconds);
 
-                    AddUiActionEngineTelemetryProperties(actionTelemetryEvent, continueAfterPreview, acceptedLicense, userAction, existingPackages, addedPackages, removedPackages, updatedPackagesOld, updatedPackagesNew);
+                    var nuGetUI = uiService as NuGetUI;
+                    AddUiActionEngineTelemetryProperties(
+                        actionTelemetryEvent,
+                        continueAfterPreview,
+                        acceptedLicense,
+                        userAction,
+                        nuGetUI?.SelectedIndex,
+                        nuGetUI?.RecommendedCount,
+                        nuGetUI?.RecommendPackages,
```

Comment:
Actually, this is a boolean indicating that this is a situation where we would have asked for recommendations independent of A/B testing status. For example whether or not there is a search string or if the package source is nuget.org. This is so that we can make correct comparisons for the A/B tests.

Reply:
Ahh...makes sense.